src/contracts/treasury/Collector.sol

```diff
@@ -205,17 +221,22 @@ contract Collector is VersionedInitializable, ICollector, ReentrancyGuard {
   }
 
   /// @inheritdoc ICollector
-  function setFundsAdmin(address admin) external onlyFundsAdmin {
-    _setFundsAdmin(admin);
+  function setACLManager(address manager) external onlyFundsAdmin {
```

Comment:
this doesn't seems so legit imo - one fundsAdmin being able to change the overall permission management.

It is probably better to make it immutable and pass on construction. If it ever needs to be changed (which i doubt), can be upgraded. There alternative would be sth like ownable, but with `VersionedInitializable` adding at this point would be quite some pain.